src/linter/block_linter.go

```diff
@@ -675,6 +679,38 @@ func (b *blockLinter) checkArray(arr *ir.ArrayExpr) {
 	}
 }
 
+func (b *blockLinter) addFixForMultilineArrayTrailingComma(item *ir.ArrayItemExpr) {
+	if !b.walker.r.config.ApplyQuickFixes {
+		return
+	}
+
+	from := item.Position.StartPos
+	to := item.Position.EndPos
+	have := b.walker.r.file.Contents()[from:to]
+
+	b.walker.r.ctx.fixes = append(b.walker.r.ctx.fixes, quickfix.TextEdit{
+		StartPos:    item.Position.StartPos,
+		EndPos:      item.Position.EndPos,
+		Replacement: string(have) + ",",
+	})
+}
+
+func (b *blockLinter) checkMultilineArrayTrailingComma(multiline bool, currentItemIndex int, items []*ir.ArrayItemExpr) {
+	if !multiline || currentItemIndex != len(items)-1 {
+		return
+	}
+
+	item := items[currentItemIndex]
+
+	from := item.Position.StartPos
+	to := item.Position.EndPos
+	itemText := b.walker.r.file.Contents()[from : to+1]
```

Comment:
You don't check that there is enough length in the contents. The code might panic.

Reply:
Done.